data/faq/application_template.md

```diff
@@ -0,0 +1,122 @@
+For example, to list your contributions to coala you can mention:
+
+- https://github.com/coala/coala/commits/master?author=YOUR_USER_NAME
+  for coala repo
+- https://github.com/coala/coala-bears/commits/master?author=YOUR_USER_NAME
+  for coala-bears repo
+
+You may also mention specific issues/commits using the commit id if you want
+us to take a look at that.
+
+You will have to finish the coala newcomer guide to be accepted:
+http://coala.io/newcomer
+
+
+Project Info
+------------
+
+- Which project from http://projects.coala.io are you applying for?
+  If you have your own idea, please add it to our projects list:
+  https://github.com/coala/projects/blob/master/data/projects.js
+
+- How is your project helping coala and its community?
+
+- What is the final goal for this project? What would make it a total
+  and perfect success?
+
+- Are you already engaged with the project's possible mentors and do
+  you have any preference for a particular mentor?
+
+- What parts of coala do you have to work with in order to complete
+  this project? What else are you planning on using?
+
+- Why are you the right person to work on this project?
+
+- How do you plan to achieve completion of your project?
+    - Please provide a schedule with dates and important
+      milestones/deliverables (preferably in two week increments).
+      This is a great help in evaluating your own progress and helps you
+      achieving your goal as it forces you to plan your work preemptively.
+
+Add all ideas you have to solve the project here. This could include
+research on linters, libraries and similar problems as well as mockups
+of code, api and ui changes.
+
+
+Other Commitments
+-----------------
+
+- Do you have any other commitments during the GSoC period,
+  May 8th to August 29th?
+
+    - We don't penalize students for needing adjustments to schedule if
+      they're up-front about them and have a plan to mitigate any issues.
+      However, we *will* fail students for lying about their availability
+      and subsequently falling behind in their work. Be honest!
+
+    - Do you have exams or classes that overlap with this period?
+
+    - Do you have plans to have any other job or internship during this
+      period?
+        - This is __NOT RECOMMENDED AT ALL__ as GSoC is intended to be a
+        full time job. But can be worked around if it only a small overlap.
+
+    - Do you have any other short term commitments during this period?
+        - e.g. family wedding, conferences, volunteer projects, planned
+          vacation days
+
+- Have you applied with any other organizations? If so, do you have a
+  preferred project/org? (This will help us in the event that more than one
+  org decides they wish to accept your proposal.)
+
+
+Extra Information
+-----------------
+
+This additional information isn't needed by coala but can help us to learn
+more about you. All fields in this section are optional.
+
+- Link to resume:
```

Comment:
resume's are horrible to write, and read, especially for someone who English is their second language.  We dont want to make a second year student feel they need to write a resume to pretend they have done a lot in their life.  They havent.  Their resume will be empty, either literally or it will be full of bullshit we need to avoid, or full of region specific references that we wont understand. (if they have international recognition already, we wont need their resume to find it).

Instead, I would love to see them build their OpenHub profile https://www.openhub.net/